playbooks/roles/ims_structured_call_interface/tasks/main.yml

```diff
@@ -95,26 +95,26 @@
         zos_operator:
           cmd: 's {{ DFS_IMS_SSID }}SCI'
           verbose: true
+      # THIS WILL NOT WORK UNTIL ZOAU 1.3 IS AVAILALBE ON ADCD
+      # - wait_for:
+      #     timeout: 10
 
-      - wait_for:
-          timeout: 10
+      # - zos_job_output:
+      #     job_name: '{{ DFS_IMS_SSID }}SCI'
+      #   register: output
 
-      - zos_job_output:
-          job_name: '{{ DFS_IMS_SSID }}SCI'
-        register: output
+      # - set_fact:
+      #     sci_output: '{{ output.jobs[0].ret_code.msg }}'
 
-      - set_fact:
-          sci_output: '{{ output.jobs[0].ret_code.msg }}'
+      # - block:
 
-      - block:
+      #   # Start SCI
+      #   - name: Start SCI
+      #     zos_operator:
+      #       cmd: 's {{ DFS_IMS_SSID }}SCI'
+      #       verbose: true
 
-        # Start SCI
-        - name: Start SCI
-          zos_operator:
-            cmd: 's {{ DFS_IMS_SSID }}SCI'
-            verbose: true
-
-        when: sci_output != 'AC'
+      #   when: sci_output != 'AC'
```

Comment:
Makes sense. Should we open an issue with a TODO item to remember to uncomment and test this code path once support for ZOAU 1.3 or newer is added?

Reply:
Definitely! Let me check with @adn1107 to see if there's already an item for this. If not I'll create it :)